src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/DotNetCoreProjectCompatibilityDetector.cs

```diff
@@ -228,6 +230,17 @@ private async Task WarnUserOfIncompatibleProjectAsync(CompatibilityLevel compatL
                     if (settingsManager != null)
                     {
                         suppressPrompt = settingsManager.GetValueOrDefault(SuppressDotNewCoreWarningKey, defaultValue: false);
+
+                        // Do not warn users about pre-release versions if they have checked "Use Preview SDKs"
+                        suppressPrompt |= settingsManager.GetValueOrDefault(UsePreviewSdkKey, defaultValue: false);
```

Comment:
I don't think this is what we want here in general.  I think we still want to be able to guide people to use something newer, even if they have checked the checkbox.